index.html

```diff
@@ -3,44 +3,179 @@
 
 <head>
     <link rel="stylesheet" type="text/css" href="css/style.css">
-    <script src="js/query.js"></script>
+    <script type="text/javascript" src="js/query.js"></script>
     <title>Genotify</title>
 </head>
 
 <body>
     <div class="header">
         <h2 name="genotifyheader">Genotify</h2>
     </div>
-    <div class="content">
-        <div class="queryform">
-            <form>
+    <div class="content flex-container">
+        <div class="splitDiv">
+            <div class="queryform">
```

Comment:
I think HTML/CSS uses "caterpillar-casing". Either switch to using caterpillar-casing for your classes and IDs, or just stick to one format.